core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -23,8 +23,10 @@ import scala.reflect.ClassTag
 import com.esotericsoftware.kryo.Kryo
 import org.scalatest.FunSuite
 
-import org.apache.spark.{SparkConf, SharedSparkContext}
+import org.apache.spark.scheduler.HighlyCompressedMapStatus
 import org.apache.spark.serializer.KryoTest._
+import org.apache.spark.storage.BlockManagerId
+import org.apache.spark.{SharedSparkContext, SparkConf}
```

Comment:
@squito For this one I think it is fine to merge, but for future ones, this line should go above all other subpackages. We do have some violations of them in the codebase, but that's the general guideline.